### PR TITLE
Document adding a CommCare build to a server

### PR DESCRIPTION
**Pull request by emord**
##### SUMMARY
Document adding a CommCare build to a server
##### ENVIRONMENTS AFFECTED
none
##### ISSUE TYPE
- Docs Pull Request

### CODE REVIEW

File: docs/setup/new_environment.md [hunk 1]
```diff
@@ -289,3 +289,9 @@ $ commcare-cloud monolith django-manage make_superuser {email}
 ```
 
 where `{email}` is the email address you would like to use as the username.
+
+## Step 13: [Optional] Add a new build
+
+To make a build of an application in CommCare, you will need to add a CommCare build to your server.
+
+To add a build to your server visit `/builds/edit_menu` and follow the instructions under "Import a new build from the build server"
```
Comment (emord) [inline docs/setup/new_environment.md]: I went through your docs and didn't find any examples of you using a URL for their server. Wasn't sure if this notation would be better than `<url>/builds/edit_menu` or something else? Do you have any preference on how we link to self hosted pages?

File: docs/setup/new_environment.md [hunk 2]
```diff
@@ -289,3 +289,9 @@ $ commcare-cloud monolith django-manage make_superuser {email}
 ```
 
 where `{email}` is the email address you would like to use as the username.
+
+## Step 13: [Optional] Add a new build
+
+To make a build of an application in CommCare, you will need to add a CommCare build to your server.
```
Comment (proteusvacuum) [inline docs/setup/new_environment.md]: This is a very confusing sentence... `to make a build... add a build`. 

```suggestion
In order to create new versions of applications created in the CommCareHQ app builder, you will need to add the latest `CommCare Mobile` and `CommCare Core` builds to your server.
```

File: docs/setup/new_environment.md (hunk 1 above)
Comment (proteusvacuum) [reply docs/setup/new_environment.md]: I think `<server url>` or something similar is fine.

Comment (emord) [reply docs/setup/new_environment.md]:
```suggestion
To add a build to your server visit `<server url>/builds/edit_menu` and follow the instructions under "Import a new build from the build server"
```

File: docs/setup/new_environment.md (hunk 2 above)
Comment (emord) [reply docs/setup/new_environment.md]: Agreed. Sometimes it feels like we could replicate https://en.wikipedia.org/wiki/Buffalo_buffalo_Buffalo_buffalo_buffalo_buffalo_Buffalo_buffalo with our usages of apps and builds

**Comment by emord** [comment]
Farid's on parental leave for the next year, so going to merge

**Pull request merged**